README.md

```diff
@@ -88,6 +88,13 @@ onPinchZoom({ scale }: VisualViewport) {
 }
 ```
 
+5. iOS currently doesn't support the `contextmenu` event. Instead, you can use a custom `longtap` event. This event
+   captures the `contextmenu` event on non-iOS devices and simulates similar behavior on iOS devices.
+
+```html
+<div (longtap)="showContextMenu($event.detail.clientX, $event.detail.clientY)">Div with context menu</div>
```

Comment:
@waterplea should it be as a plugin instead of directive in taiga-ui?

`<div (tuiLongtap)="showContextMenu($event.detail.clientX, $event.detail.clientY)">Div with context menu</div>`

Reply:
This is a workaround for iOS not properly dispatching `contextmenu` event, so I think it's a good idea to handle this as an event, not with directive, and as part of our supercharging events library so that it can be used even without Taiga UI.